### PR TITLE
bundle: update csv with info required for release

### PR DESCRIPTION
**Pull request by leelavg**
CSV description and annotations required for the release are updated

### CODE REVIEW

**Comment by leelavg** [comment]
/hold

for some clarifications on labels & valid-subscription.

File: bundle/manifests/ocs-client-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -24,7 +36,9 @@ spec:
       kind: StorageClient
       name: storageclients.ocs.openshift.io
       version: v1alpha1
-  description: foo
+  description: The OpenShift Data Foundation client operator connects to a centralised
+    storage provided by OpenShift Data Foundation provider cluster to serve storage
+    for workloads.
```
Comment (nb-ohad) [inline bundle/manifests/ocs-client-operator.clusterserviceversion.yaml:41]:
```suggestion
  description: OpenShift Data Foundation client operator enables consumption of storage services from a remote centralized OpenShift Data Foundation provider cluster 
```

**Review by nb-ohad — CHANGES_REQUESTED** [review]

File: bundle/manifests/ocs-client-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (leelavg) [reply bundle/manifests/ocs-client-operator.clusterserviceversion.yaml:41]: done.

**Comment by leelavg** [comment]
/hold

for a decision on suggested namespace annotation.

**Comment by nb-ohad** [comment]
\lgtm

**Pull request merged**